feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/BottomSheetDaySessionsFragment.kt

```diff
@@ -129,6 +129,7 @@ class BottomSheetDaySessionsFragment : DaggerFragment() {
                 count as Int
             )
             binding.filteredSessionCount.isVisible = uiModel.filters.isFiltered()
+            binding.startFilter.text = if (uiModel.filters.isFiltered()) getString(R.string.filter_now) else getString(R.string.start_filter)
```

Comment:
How about this?
val startFilterTextRes =  if (uiModel.filters.isFiltered()) R.string.filter_now else R.string.start_filter
binding.startFilter.text = getString(startFilterTextRes)

Reply:
Good, I fix this.